data/members.ts

```diff
@@ -7,419 +7,335 @@ export type PersonInfoProps = {
 };
 
 const members: PersonInfoProps[] = [
+  {
+    name: "Jason An",
+    role: "Co-President",
+    bio: "2nd year Computer Science major",
+    pronouns: "he/him",
+    photo: "jason.png"
+  },
+  {
+    name: "Savannah Alanis",
+    role: "Co-President",
+    bio: "2nd year Computer Science & Linguistics major",
+    pronouns: "she/her",
+    photo: "savannah.jpg"
+  },
   {
     name: "Benson Liu",
-    role: "Co-President, Psi Beta Rho Director",
+    role: "Advisor",
     bio: "Hey, I'm Benson! I'm a 3rd year Computer Science major from New Jersey who enjoys hacking & playing CTFs! In my free time, I like to watch anime or baseball (my favorite team is the Yankees).",
     pronouns: "he/him",
-    photo: "bliu.png",
+    photo: "bliu.png"
   },
```

Comment:
Can you move Benson and Benson only down 🤡.

His lengthy bio causes the whole next row of officers to be yeeted from the initial viewport. Also, I wanna see if he will make another pr to bring it back lmao.